api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -709,6 +709,7 @@ message Cluster {
   EdsClusterConfig eds_cluster_config = 3;
 
   // The timeout for new network connections to hosts in the cluster.
+  // If not set, a default value of 5s will be used.
```

Comment:
@mk46 actually, can you make sure @wrowe concerns are met? I think https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto.html is handled automatically by this, but there was the ask on FAQ.

Also, maybe add a version history entry. Thanks.

Reply:
added in version history and FAQ.